### PR TITLE
Fix docs warnings

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: sumpy/point_calculus.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def laplace(self, f_values):
     def div(self, arg):
         """
         :arg arg: an object array containing
-            :class:`numpy.ndarrays` with shape ``(npoints_total,)``.
+            :class:`numpy.ndarray` s with shape ``(npoints_total,)``.
```
Comment (inducer) [inline sumpy/point_calculus.py:216]:
```suggestion
            :class:`numpy.ndarray`\ s with shape ``(npoints_total,)``.
```

File: sumpy/point_calculus.py [hunk 2]
```diff
@@ -225,7 +225,7 @@ def curl(self, arg):
         """Take the curl of the vector quantity *arg*.
 
         :arg arg: an object array of shape ``(3,)`` containing
-            :class:`numpy.ndarrays` with shape ``(npoints_total,)``.
+            :class:`numpy.ndarray` s with shape ``(npoints_total,)``.
```
Comment (inducer) [inline sumpy/point_calculus.py:228]:
```suggestion
            :class:`numpy.ndarray`\ s with shape ``(npoints_total,)``.
```

File: sumpy/point_calculus.py [hunk 3]
```diff
@@ -213,7 +213,7 @@ def laplace(self, f_values):
     def div(self, arg):
         """
```
Comment (inducer) [inline sumpy/point_calculus.py:214]:
```suggestion
        r"""
```

**Comment by isuruf** [comment]
Any idea why we see `WARNING: py:mod reference target not found: sumpy`? @kaushikcfd saw the same in loopy

**Comment by inducer** [comment]
@isuruf Just add a
```
.. module:: sumpy
```
somewhere sensible (maybe on the main page?).

**Comment by isuruf** [comment]
One final error I can't figure out. `/home/runner/work/sumpy/sumpy/sumpy/toys.py:docstring of sumpy.toys.ExpansionPotentialSource:7: WARNING: py:meth reference target not found: matplotlib.pyplot.annotate`

**Comment by kaushikcfd** [comment]
> One final error I can't figure out. `/home/runner/work/sumpy/sumpy/sumpy/toys.py:docstring of sumpy.toys.ExpansionPotentialSource:7: WARNING: py:meth reference target not found: matplotlib.pyplot.annotate`

Probably `matplotlib.pyplot.annotate` is `:func:` and not `:meth:`?

**Comment by isuruf** [comment]
Yes, it was. Thanks.

**Comment by inducer** [comment]
This looks great! Thanks for working on it.

**Pull request merged**